ready-to-use fields/README.md

```diff
@@ -0,0 +1,15 @@
+<img align="right" src="https://cdn.miranj.in/mc3/img/apple-touch-icon-precomposed.png" width="57" height="57" alt="Miranj">
+
+# Miranj Craft Boilerplate
+
+A bunch of ready to use fields provided by Miranj to speed-up your project development process. It uses the [Field Manager] plugin (https://github.com/verbb/field-manager) to import the fields.
```

Comment:
Markdown link formatting is off. See the rendered preview.

Reply:
Fixed.